### PR TITLE
feat: arbitrum one docs

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Update docs regarding Arbitrum one and remove Goerli references.
Also added arb1 endpoint to blockscout config.

# Changes

- [x] Add arbitrum one to ordebook openapi
- [x] Remove goerli from ordebook openapi
- [x] Sort orderbook openapi urls by chain type (mainnet, side chains, l2s, test chains)
- [x] Same thing, but for README
- [x] Add arb1 endpoint to blockscout config

## How to test

1. Check readme
2. Check api docs

### CODE REVIEW

**Review by squadgazzz — APPROVED** [review]
Cool, thanks!

File: crates/shared/src/bad_token/token_owner_finder/blockscout.rs
Comment (squadgazzz) [reply crates/shared/src/bad_token/token_owner_finder/blockscout.rs:25]: `https://arbitrum.blockscout.com/api` should be probably used for the arb1 since a tokenOwnerFinder is configured: https://github.com/cowprotocol/infrastructure/pull/1535/files#diff-9056c74c0da2da5bf9f902937af7d93fc926f33b2142e30b154dbbea16a4e825R91

Comment (alfetopito) [reply crates/shared/src/bad_token/token_owner_finder/blockscout.rs]: Thank you!

https://github.com/cowprotocol/services/pull/2757/commits/f753b0eb48bc885ab75f2065bb92ff90f5d9b4f2

Comment (squadgazzz) [reply crates/shared/src/bad_token/token_owner_finder/blockscout.rs]: Nice, let's wait for Martin's approval before merging it.

Comment (alfetopito) [reply crates/shared/src/bad_token/token_owner_finder/blockscout.rs]: Ops, sorry. I turned on the auto-merge and your approval was enough for it to trigger 😬 

@MartinquaXD let me know whether you want me to revert or do any changes

Comment (MartinquaXD) [reply crates/shared/src/bad_token/token_owner_finder/blockscout.rs]: No problem. Thanks for the PR. 🙌

**Pull request merged**
